articles/libraries/auth0-swift/index.md

```diff
@@ -28,19 +28,17 @@ To install it, simply add the following line to your Podfile:
 pod 'Auth0', '~> 1.0'
 ```
 
-### Carthage
+### Using Carthage
 
 In your Cartfile add this line
 
 ```
 github "auth0/Auth0.swift" ~> 1.0
 ```
 
-## Usage
+## Auth0.plist
 
-### Auth0.plist
-
-To avoid specifying clientId & domain you can add a `Auth0.plist` file to your main bundle. Here is an example of the file contents:
+To avoid specifying your clientId and domain in-line you should add an `Auth0.plist` file to your main bundle. Here is an example of the file contents:
```

Comment:
I think in Auth0.swift vs Lock you have to use `Auth0.plist` for credentials, it's a better place to put it anyway. I would word this as being the way vs avoiding another way.